documentation/src/test/java/org/hibernate/search/documentation/search/projection/ProjectionDslIT.java

```diff
@@ -452,7 +452,11 @@ public void composite() {
 			List<Object[]> hits = searchSession.search( Book.class )
 					.select( f -> f.composite() // <1>
 							.from( f.field( "title", String.class ), // <2>
-									f.field( "genre", Genre.class ) ) // <3>
+									f.field( "genre", Genre.class ),
+									f.highlight( "title" )
+											.unified()
+											.tag( "<strong>", "</strong>" )
+							) // <3>
```

Comment:
Are you sure this is wise? I wouldn't want to complicate unrelated documentation unnecessarily.